python/ray/_private/test_utils.py

```diff
@@ -1737,3 +1737,12 @@ def get_gcs_memory_used():
     }
     assert "gcs_server" in m
     return sum(m.values())
+
+
+def wandb_populate_run_location_hook():
+    """
+    Example external hook to populate W&B project and group env vars in
+    WandbIntegrationTest.testWandbLoggerConfig
+    """
+    os.environ["WANDB_PROJECT_NAME"] = "test_project"
+    os.environ["WANDB_GROUP_NAME"] = "test_group"
```

Comment:
Should we import `WANDB_PROJECT_ENV_VAR` and `WANDB_GROUP_ENV_VAR` here? (locally)

Reply:
I was seeing a `ModuleNotFoundError: No module named 'pandas'` error on the minimal install tests (https://buildkite.com/ray-project/oss-ci-build-pr/builds/7359#01850810-6f61-4017-82c0-f953782b0e9b) when I tried that previously because the `test_utils` files was importing various files from `tune`

Reply:
Does this happen also with local imports in the function?